### ISSUE TITLE
Popup with Formidable Forms doesn't show reCaptcha 2

### ISSUE THREAD

**Issue by tamelander** [elementor/elementor#8983]
## Prerequisites

- [x ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ x] The issue still exists against the latest stable version of Elementor.
*** I am using 7.2 beta 2, which should contain the latest stable release for popup ***

## Description

I am building a popup with a Contact form. I use Formidable Pro forms with reCaptcha 2. In a regular page the reCaptcha shows and works. But not in a popup when viewing the page. Strange thing is when I preview the popup from the Elementor popup editing window, the reCaptcha shows. Could that be a hint? For that reason I didn't follow recommended steps with disabling every other plugin and changing the theme to TwentyNineteen.

**Comment by RonR-WebDesign** [comment]
I can confirm this issue.  [This might help](https://premium.wpmudev.org/forums/topic/formidable-forms-recaptcha-not-working-with-hustle-popups)

That mentions an issue with "timed activation".  I turned off the Popups Entrance Animation and that did not fix this.

**Comment by tamelander** [comment]
The Elementor popup closes on Formidable form submission but the message is not sent. It would be nice to get this to work. Popups are great for getting people to act and move forward.

**Comment by RonR-WebDesign** [comment]
...forgot to mention.  The recaptcha does work in the Popup edit Preview mode.

**Comment by tamelander** [comment]
Exactly, for me too "...Strange thing is when I preview the popup from the Elementor popup editing window, the reCaptcha shows."

**Comment by RonR-WebDesign** [comment]
I had to scrap the Elementor Popups with Formidable forms and use plugin: Popup Builder.   Works great with recaptcha but more of a pain to setup....

**Comment by tamelander** [comment]
I will await a solution. Formidable Forms is the forms builder I like best.

**Comment by amalia-llc** [comment]
Hi. I have the same issue. I'm using Formidable forms to create an email sign up form within the Elementor pop-up, but the Google ReCaptcha is not showing up. When I try to sign up anyways, it prevents me from doing so saying there's an error saying 'The captcha is missing from this form'. Is this an Elementor issue with the pop-up? 

Any progress on a solution? Thanks!

**Comment by RonR-WebDesign** [comment]
amalia-llc :: As I mentioned above, it is broken and I moved to plugin: Popup Builder.  

Elementor does not seem to care about bugs.  This is the second that I've found with zero interest from the Elementor team.  We have to manage our own work-arounds.... sigh...

**Comment by jackmcpickle** [comment]
Pretty simple. just retrigger the form recatpcha when the elementor modal loads. should work for all recaptcha types. (google, cloudflare,etc).

Just add to theme or load via a js file. 

```js
jQuery(document).on('elementor/popup/show', () => {
	setTimeout(() => {
		const captchaField = document.querySelector('.cf-turnstile');
		window.frmFrontForm.renderCaptcha(captchaField);
	}, 100);
} );
```
